source/content/guides/composer-convert.md

```diff
@@ -1,18 +1,29 @@
 ---
 title: Convert a Standard Drupal 8 Site to a Composer Managed Site
-description: Drupal 8 sites often require the usage of Composer to manage site dependencies. The need to begin using Composer for a site build can often surface after a site is in development, necessitating a divergence from the Pantheon managed upstream.
+description: Upgrade a Drupal 8 site to integrated Composer and Drupal 9.
 type: guide
 permalink: docs/guides/:basename
 cms: "Drupal 8"
 categories: [develop]
 tags: [composer, site, workflow]
-contributors: [dustinleblanc]
+contributors: [dustinleblanc, greg-1-anderson]
+reviewed: "2020-12-01"
 ---
 
+Drupal sites often require the use of Composer to manage site dependencies. The need to begin using Composer for a site build can often surface after a site is in development, necessitating a conversion of the site's codebase. At a high level, the goals of the conversion are to remove dependencies that Composer will manage from your Git repository, and tell Composer about those dependencies instead.
+
+Existing sites that wish to upgrade from Drupal 8 to Drupal 9 and convert to Integrated Composer, or just convert to Integrated Composer and defer the Drupal 9 upgrade will need to go through some manual steps.
+
 ## Before You Begin
 
 - Review our documentation on [Git](/git), [Composer](/composer), and [Terminus](/terminus), and have them installed and configured on your local computer.
-- [Clone](/git#clone-your-site-codebase) your _current_ Pantheon site repository to a working directory on your local computer.
+   - Mac users can use [Homebrew](https://brew.sh/) to install both Git and Composer, along with their required dependencies:
+
+     ```bash{promptUser:user}
+     brew install git composer
```

Comment:
It might also be a good idea to advise users that they should be on Composer 2. If we do this, then we should probably also link to a document with guidance. If we don't do this, they may run into problems if they push a site that only works with Composer 1 up to the platform, which uses Composer 2.